lib/rules/react-svg-path-precision.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
                 }
             }
         }
-        const pattern = `(\\.${"[0-9]".repeat(precision)})[0-9]+`;
+        const pattern = `\\d*\\.${"\\d+".repeat(precision)}\\d+`;
```

Comment:
You could also write this as: `\\d*\\.\\d{${precision},}` (so roughly: `/\d*\.\d{3,}` (which means at least 3 matches)

I wonder if the first `\\d*` needs to be a `\\d+` in order to fix the leading 0 issue.

Reply:
Neat.  I'll give that a try.